RELEASE.md

```diff
@@ -1,6 +1,57 @@
 # Release 1.6.0
 
-The 1.6 minor series tracks TensorFlow 1.6. Release notes upcoming.
+NOTICE: TensorBoard 1.6.0+ has moved to the `tensorboard` package name on PyPI:
```

Comment:
Optional nit: in spirit of go/java-practices/javadoc IIRC? consider `**Note:**`

Reply:
Ack.  I already used NOTICE for 1.5.1 relnotes, which were approved twice already, and the PyPI description for 1.5.1, and I think it's better to be consistent.